File: tjhsst comp sci 2/Java/Units/Unit1/Climber.java
//Name: Vivian Feng and Shriya Muthukumar Date: March 16.2020

import edu.fcps.karel2.Robot;
import edu.fcps.karel2.Display;

public class Climber extends Athlete
{
   public Climber()
   {
      super();
   }
   public Climber(int x)
   {
      super(x,1,Display.NORTH,1);
   }
   public void climbUpRight()
   {
      //tL,m,m,tR,m
      turnLeft();
      move();
      move();
      turnRight();
      move();
      
   }
   public void climbDownRight(){
      move();
      turnRight();
      move();
      move();
      turnLeft();
     }
     
  public void climbUpLeft(){
   turnRight();
   move();
   move();
   turnLeft();
   move();
  }
  public void climbDownLeft(){
   move();
   turnLeft();
   move();
   move();
   turnRight();
   
  }
   
}

File: tjhsst comp sci 2/pyKarel Student Code/Lab03 Patched/Climber.py
#Name:Karthik and Vivian
#Date:09/04/2019

from pyKarel import *

from Athlete import *

class Climber(Athlete):
   def __init__(self,world,x,y=1,direction=north,beepers = 1):
      Athlete.__init__(self,world,x,y,direction,beepers)
   
   def climbUpRight(self):
      self.turnLeft()
      self.move()
      self.move()
      self.turnRight()
      self.move()
      
      
      
   def climbDownRight(self):
      self.move()
      self.turnRight()
      self.move()
      self.move()
      self.turnLeft()
      
   def climbUpLeft(self):
      self.turnRight()
      self.move()
      self.move()
      self.turnLeft()
      self.move()
      
   def climbDownLeft(self):
      self.move()
      self.turnLeft()
      self.move()
      self.move()
      self.turnRight()

   
      
     
